### PR TITLE
[WA-4.3]-Display advertisements

### PR DESCRIPTION
**Pull request by Aleksandra988**

### CODE REVIEW

File: WebApplication/Backend/DTO/SurveyDTO.cs [hunk 1]
```diff
@@ -10,86 +10,89 @@ public class SurveyDTO
...
-        private string question16;
-        private string question17;
-        private string question18;
-        private string question19;
-        private string question20;
-        private string question22;
-        private string question21;
-        private string question23;
+        private List<String> questions;
+        //private string question1;
```
Comment (milicasiriski) [inline WebApplication/Backend/DTO/SurveyDTO.cs]: If refactoring is complete you can delete this 😄

File: WebApplication/Backend/Services/AppointmentsService.cs [hunk 2]
```diff
@@ -1,82 +1,82 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using HealthClinicBackend.Backend.Dto;
-using HealthClinicBackend.Backend.Model.Accounts;
-using HealthClinicBackend.Backend.Model.Schedule;
-using WebApplication.Backend.Repositorys;
-using WebApplication.Backend.Repositorys.Interfaces;
+﻿//using System;
```
Comment (milicasiriski) [inline WebApplication/Backend/Services/AppointmentsService.cs:1]: If this is unused you can delete it

File: WebApplication/Backend/Util/UtilDbContext.cs [hunk 3]
```diff
@@ -211,7 +211,7 @@ private static void PatientCreation(ModelBuilder modelBuilder)
         }
 
         private static void SurveyCreation(ModelBuilder modelBuilder)
-        {
+        {/*
```
Comment (milicasiriski) [inline WebApplication/Backend/Util/UtilDbContext.cs]: delete if unused

File: WebApplicationTests/SurveyAndStatisticsServiceTests.cs [hunk 4]
```diff
@@ -13,7 +13,7 @@ namespace WebApplicationTests
     public class SurveyAndStatisticsServiceTests
     {
         Survey surveyTest = new Survey
-               ("001", "Bozo Bozic", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", new DateTime(2017, 05, 17));
+               ("001", "Bozo Bozic", /*"3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", */new DateTime(2017, 05, 17));
```
Comment (milicasiriski) [inline WebApplicationTests/SurveyAndStatisticsServiceTests.cs]: delete unnecessary comments

**Review by milicasiriski — APPROVED** [review]
Everything looks good, just delete the leftover comments

**Review by Tanjaa1 — APPROVED** [review]

**Pull request merged**